zc_common/remote_resource/request.py

```diff
@@ -25,14 +25,7 @@ class RouteNotFoundException(Exception):
 
 
 class ServiceRequestException(Exception):
-    """An exception commonly thrown when an HTTP request to another service endpoint fails."""
-    message = None
-    response = None
-
-    def __init__(self, message, response):
```

Comment:
@ograycode Almost everything in the response is in the message, so I prefer not to mess with it and stick to a simple inheritance. This is an internal error, so no big deal keeping them. Passing the object could also be problematic, since we might have issues for serializing, etc. IMO lets keep it simple!